packages/fuchsia_ctl/tool/build.sh

```diff
@@ -25,7 +25,7 @@ mkdir -p build
 
 tool/dart-sdk/bin/pub get
 
-tool/dart-sdk/bin/dart2aot bin/main.dart build/main.aot
+tool/dart-sdk/bin/dart2native -k aot bin/main.dart -o build/main.aot
 rm -f build/main.aot.dill
 cp -f tool/dart-sdk/bin/dartaotruntime build/
```

Comment:
Do we still need `dartaotruntime`?

Reply:
No, dart2native should produce a single binary

Reply:
In that case I think we want to name it differently.  We can probably just ditch the wrapper script as well.

Reply:
Done!  updated the script to create a native binary and deleted the wrapper script.

Task execution using led:

https://chromium-swarm.appspot.com/task?id=48beb14ee4497310